src/devices/Common/Iot/Device/Common/SimpleConsoleLoggerFactory.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
```

Comment:
1. missing copyright header
2. should this be showed in the sample rather than part of the library? (I'd suggest to use root level sample folder to showcase logging)

Reply:
Yea, I need to add a readme

Reply:
Added documentation on usage. We can discuss, but we might want to keep this in the library, to avoid code duplication in all our examples when more and more bindings support logging.